src/web/components/PulsingDot.tsx

```diff
@@ -23,23 +16,36 @@ const pulsingDot = (colour?: string) => css`
 		content: '';
 		margin-right: 0.1875rem;
 		vertical-align: initial;
-		animation: ${livePulse} 1s infinite;
 	}
 `;
 
+const livePulse = keyframes`{
+    0% {opacity: 1;}
+    10% {opacity: .25;}
+    40% {opacity: 1;}
+    100% {opacity: 1;}
+}`;
+
+const animate = css`
+	animation: ${livePulse} 1s infinite;
+`;
+
 interface Props {
 	colour?: string;
 }
 
 export const PulsingDot = ({ colour }: Props) => {
 	// Respect the accessibility flag set here
 	// https://www.theguardian.com/help/accessibility-help
-	const flashingIsDisabled = !!document.getElementsByClassName(
-		DISABLE_FLASHING_ELEMENTS_CLASS,
-	).length;
-	if (flashingIsDisabled) {
-		return null;
-	}
-
-	return <span className={pulsingDot(colour)} />;
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+	const flashingPreference = storage.local.get(
+		'gu.prefs.accessibility.flashing-elements',
+	);
+	// flashingPreference is null if no preference exists and explicitly
+	// false when the reader has said they don't want flashing
+	const flashingEnabled = flashingPreference !== false;
+	console.log('flashingEnabled', flashingEnabled);
+	return (
+		<span className={cx(dotStyles(colour), flashingEnabled && animate)} />
```

Comment:
Wondering if we should also use the opportunity to respect -prefers-reduced-motion? Either via the media query (and refactor the css bit) or implement a JS helper that you can then pass in as here, something like:

```js
function getPrefersReducedMotion() {
  const QUERY = '(prefers-reduced-motion: no-preference)';
  const mediaQueryList = window.matchMedia(QUERY);
  const prefersReducedMotion = !mediaQueryList.matches;
  return prefersReducedMotion;
}
```

https://www.joshwcomeau.com/react/prefers-reduced-motion/#in-js-land

Reply:
Great idea! I'll add this 👍

Reply:
I added support for `prefers-reducced-motion` using the media query in the css directly as that seems to work well and is simple

Reply:
❤️